packages/arb-token-bridge-ui/src/components/TransferPanel/useTransferReadiness.ts

```diff
@@ -90,6 +91,7 @@ export function useTransferReadiness({
     erc20: [erc20L2Balances]
   } = useBalance({ provider: l2Provider, walletAddress })
   const { error: destinationAddressError } = useDestinationAddressStore()
+  const { gasSummary } = useGasSummaryStore()
```

Comment:
I get the point of encapsulating it within the hook, so we don't need to repeat the call every time we need `useTransferReadiness`, but what about testing? Can we easily mock it in the future if we want to add unit test for this hook? We have this pattern on multiple hooks, so just wondering what we want to do

Reply:
as discussed, we can implement SWR on a later PR